plugins/code-coverage-backend/src/service/standaloneServer.ts

```diff
@@ -54,7 +54,7 @@ export async function startStandaloneServer(
 
   logger.debug('Starting application server...');
   const router = await createRouter({
-    database: { getClient: async () => db },
+    database: { getClient: async () => db, migrations: { apply: true } },
```

Comment:
We should be able to avoid this if we make migrations and apply/skip optional right?